deploy-edgecluster/render_edgeclusters.sh

```diff
@@ -290,6 +290,28 @@ spec:
          auto-gateway: true
          auto-routes: true
        mtu: 1500
+       mac-address: '$CHANGE_EDGE_MASTER_MGMT_INT_MAC'
+EOF
+        echo ">> Checking Number of Interfaces"
+        echo "Edge-cluster: ${cluster}"
+        echo "Master: ${master}"
+        if [[ ${CHANGE_EDGE_MASTER_PUB_INT_MAC} == "null" ]]; then
+          cat <<EOF >>${OUTPUT}
+     - name: $CHANGE_EDGE_MASTER_MGMT_INT.102
+       type: vlan
+       state: up
+       vlan:
+         base-iface: $CHANGE_EDGE_MASTER_MGMT_INT
+         id: 102
```

Comment:
Shouldn't this ID be configurable to avoid clashes?

Reply:
this is for the internal network and the traffic is between chassis nodes (or vm) we're configuring... As we are not using more vlan id in our deployment is not possible any collision.

Reply:
remember we're using the dhcp inteface to connect to the customer/factory network, so no prob to fix it as well as we put an static network 7.0/24. This is the same case, and it's not possible any colision externally with the customer/factory network

Reply:
It may be needed in the future but let's start with this as-is for now.